kurtosis/beaconkit-all.yaml

```diff
@@ -93,7 +93,7 @@ node_settings:
       max_memory: 2048
     images:
       besu: hyperledger/besu:24.5.4
-      erigon: thorax/erigon:v2.60.1
+      erigon: erigontech/erigon:v2.60.1
```

Comment:
The new repo has images from 2.60.8 onwards. Need to change the docker image version. Similar comments at other places as E2E tests are failing.

Reply:
thanks, bumped to the latest version wherever applicable.